rtb/analisi-requisiti/src/casi-uso.d/cliente-prenotazione.tex

```diff
@@ -1 +1,137 @@
-E
+\textbf{UC19-Prenotazione}
+\begin{itemize}
+    \item \textbf{Attore principale: }cliente.
+    \item \textbf{Precondizioni: }il cliente sta visualizzando le informazioni di un ristorante.
+    \item \textbf{Postcondizioni: }la prenotazione è salvata nella lista delle prenotazioni del cliente.
+    \item \textbf{Scenario principale:} 
+        \begin{enumerate}
+            \item Il cliente seleziona la funzionalità di richiesta di una nuova prenotazione;
+            \item Il sistema illustra i giorni in cui è possibile effettuare la richiesta;
+            \item Il cliente seleziona il giorno di calendario;
+            \item Il sistema illustra le fasce orarie disponibili;
+            \item Il cliente seleziona l'orario;
+            \item Il cliente inserisce il numero di persone che parteciperanno alla prenotazione;
+            \item Il cliente dà la conferma ed effettua la richiesta;
+            \item il cliente visualizza la lista delle sue prenotazioni,ove è presente la prenotazione appena
+            effettuata.
+        \end{enumerate}
+\end{itemize}
+
+%il cliente può visualizzare solo prenotazioni passate,quelle future...? le divido per categorie?
+%ci sta aggiungere un'estensione se non ci sono prenotazioni?tipo "il sistema comunica al cliente che la lista è vuota"
+\textbf{UC20-Visualizzazione lista prenotazioni}
+\begin{itemize}
+    \item \textbf{Attore principale: } cliente.
+    \item \textbf{Precondizioni: }il cliente sta visualizzando le sue informazioni personali.
+    \item \textbf{Postcondizioni: }il cliente visualizza la lista delle sue prenotazioni.
+    \item \textbf{Scenario principale:} 
+        \begin{enumerate}
+            \item L'utente seleziona la funzionalità di visualizzazione delle sue prenotazioni;
+            
+            \item L'utente visualizza la lista delle sue prenotazioni;
```

Comment:
si può dire: "Di default vengono visualizzate le prenotazione future più vicine" (intendo più vicine rispetto alla data di oggi (ad es. oggi è sabato mi mostra prima prenotazione di martedì che di venerdì prossimo)

Reply:
suggerisco : "prenotazioni future in ordine temporale crescente"

Reply:
Se nello stato della prenotazione possono essere indicati i vari stati di "pagata ", "in corso" ecc vuol dire che vedo anche le prenotazioni passate,presenti e future